tutorials/inputs/mouse_and_input_coordinates.rst

```diff
@@ -36,7 +36,7 @@ for example:
            print("Mouse Motion at: ", event.position)
 
        # Print the size of the viewport.
-       print("Viewport Resolution is: ", get_viewport_rect().size)
+       print("Viewport Resolution is: ", get_viewport().size)
```

Comment:
```suggestion
       print("Viewport Resolution is: ", get_viewport().get_visible_rect())
```

`get_viewport_rect()` is defined in `CanvasItem` nodes. So the previous description doesn't work for all nodes and should be updated.
However `get_viewport_rect()` doesn't return `get_viewport().size`, but `get_viewport().get_visible_rect()`, so it should be changed accordingly in order to keep the same behavior as the previous description.